docs/pages/get-started/faq.md

```diff
@@ -410,83 +423,141 @@ Currently, messaging incurs no fee. As XMTP decentralizes, messaging between par
 
 There are no messaging-related fees incurred by developers for building with the XMTP SDK.
 
-### What are the costs of XMTP message storage and retrieval?
-
-Messages are stored off-chain on the XMTP network, with all nodes currently hosted by Ephemera. Ephemera currently absorbs all message storage and retrieval costs.
-
-There are no message storage and retrieval-related fees incurred by developers for building with the XMTP SDK.
-
 ## Security
 
 ### Has XMTP undergone a security audit?
 
+A security audit of [LibXMTP](https://github.com/xmtp/libxmtp) and its use of Messaging Layer Security (MLS) will begin in Q4 2024. The audit will be performed by [NCC Group](https://www.nccgroup.com/).
+
 The [XMTP JavaScript SDK](https://github.com/xmtp/xmtp-js) (`xmtp-js`) was verified by [CertiK](https://www.certik.com/company/about) on Feb 6, 2023. Read their [XMTP security assessment](https://raw.githubusercontent.com/xmtp/docs-xmtp-org/main/docs/pages/img/REP-final-20230207T000355Z.pdf).
 
 ### How does XMTP establish a secure and fraud-proof relationship between two identities?
 
-Blockchain accounts sign and advertise a set of keys to start using XMTP. XMTP uses these keys to establish a shared secret between the blockchain accounts. It then uses the shared secret to generate a key used to encrypt an invitation that allows the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
+XMTP uses a combination of Ethereum wallet signatures and MLS (Messaging Layer Security):
```

Comment:
is this an accurate summary of how v3 works?

Reply:
So there is no identity key anymore - there are only wallets and installation keys. Also, the format of the credential has changed significantly after the multi-identity work, happy to run through it verbally!

A rough intuition:
- Each Inbox ID represents a constantly changing 'set' of identities, that could be either wallets or installations.
- The first wallet to claim an Inbox ID is the first member of the set, and is also the 'recovery address'.
- More wallets and installations can be added to the set via two-way signatures (a signature from a key currently in the set, accompanied by a signature from the key being added to the set)
- Wallets and installations can be removed from the set using a signature from the 'recovery address' (one-way signature)
- These changes to the set are published as 'identity updates' to the server, which stores the complete set of updates for any inbox ID as an 'Inbox log'.

Credentials no longer contain anything other than an Inbox ID. Whenever any other client receives your credential, they will verify it by downloading the 'Inbox log' for that Inbox ID, reading through the series of updates, and verifying at the end that your installation key is a currently valid member of the 'set'.

As an example, if you are a first time user of XMTP, these will be the signatures required:
1. Wallet signature to claim the Inbox ID
2. Wallet signature to add the installation key to the inbox (which is generated by the app)
3. Installation key signature to consent to being added

We have 'batch signatures', so a single end-user wallet signature can sign for both of the first two points. For the third signature, this happens inside the app invisibly to the user (intuitively: wallet signatures represent user actions, installation signatures represent app actions).

Reply:
What a generous comment from you, @richardhuaaa - thank you!

I have made these updates to reflect the info you provided: https://github.com/xmtp/docs-xmtp-org/pull/26/files#diff-0f5558e2ccc886a61098eac3e4020166d6c4dee87249fc7dcda8eba0f5bd1196R432

WDYT?

Beyond this first pass, I can do more to bring this FAQ in line with [Scott's excellent work here](https://docs.xmtp.org/protocol/v3/identity), which I know you have already invested time in.

Reply:
LGTM!